lib/plugins/aws/package/compile/events/kafka.js

```diff
@@ -243,6 +255,10 @@ class AwsCompileKafkaEvents {
           };
         }
 
+        if (startingPositionTimestamp) {
```

Comment:
If allowed minimum is `0`, then we need to upgrade this check to support `0`

Reply:
really good point. I'll work on some tests and checks

Reply:
Eventually decided to leave the value validation to Cloudformation instead of managing it in serverless

Reply:
Ok, but we need to be consistent with our schema. If in schema we treat `0` as valid value, then also we should pass it through to CF template.

Currently as implemented `startingPosition: 0` is treated same as if `startingPosition` was not provided at all, and that feels wrong

Reply:
Good catch!